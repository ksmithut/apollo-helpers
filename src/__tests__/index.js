/* eslint-env jest */
'use strict'

const apolloHelpers = require('../index')

describe('apolloHelpers', () => {
  test('should have methods', () => {
    expect(apolloHelpers).toMatchSnapshot()
  })
})
