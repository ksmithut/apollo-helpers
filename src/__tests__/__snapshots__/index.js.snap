// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`apolloHelpers should have methods 1`] = `
Object {
  "compileGraphqlResources": [Function],
  "createExecutor": [Function],
}
`;
